website/docs/cloud-docs/run/cli.mdx

```diff
@@ -205,20 +209,47 @@ If you prefer to use the CLI in a non-interactive environment, we recommend firs
 
 [permissions-citation]: #intentionally-unused---keep-for-maintainers
 
+## Remote Saved Plans
+
+-> **Version note:** Saved plans require at least Terraform CLI v1.6.0.
+
+In workspaces that support `terraform apply`, you also have the option of performing the plan and apply as separate steps, using the standard variations of the relevant Terraform commands:
+
+- `terraform plan -out <FILE>` will perform and save a plan.
+- `terraform apply <FILE>` will apply a previously saved plan.
+- `terraform show <FILE>` (and `terraform show -json <FILE>`) will inspect a previously saved plan.
+
+Saved plan runs are halfway between speculative plans and standard plan and apply runs. They allow you to:
+
+- Perform cheap exploratory plans, while retaining the option of applying a specific plan once you are satisfied with your work.
+- Perform other tasks in your terminal between the plan and apply stages.
+- Perform the plan and apply operations on separate machines (as is common in continuous integration workflows).
+
+Saved plans become _stale_ once the state they were planned against is no longer valid (usually due to a different run being applied). In Terraform Cloud, stale saved plan runs are automatically detected and discarded. When examining a remote saved plan, the `terraform show` command (without the `-json` option) will inform you if a plan has been discarded or is otherwise unable to be applied.
+
+### File Contents and Permissions
+
+In order to abide by Terraform Cloud's permissions model, remote saved plans do not use the same local file format as locally executed saved plans; instead, they are a thin reference to a remote run, and Terraform CLI relies on authenticated network requests to inspect and apply remote plans. This helps avoid the accidental exposure of credentials or other sensitive information.
+
+The `terraform show -json` command requires workspace admin permissions when performed with a remote saved plan; this is because the [machine-readable JSON plan format](/terraform/internals/json-format) contains unredacted sensitive information (alongside redaction hints for use by systems that consume the format). The human-readable version of `terraform show` receives pre-redacted information, so it only requires the read runs permission.
```

Comment:
This garbles the meaning somewhat, since `show` doesn't peform the plan, so:

```suggestion
The `terraform show -json` command requires [workspace admin permissions](/terraform/cloud-docs/users-teams-organizations/permissions#workspace-admins) to inspect a remote saved plan; this is because the [machine-readable JSON plan format](/terraform/internals/json-format) contains unredacted sensitive information (alongside redaction hints for use by systems that consume the format). The human-readable version of `terraform show` only requires the read runs permission, because it uses pre-redacted information.
```